requirements/lit/docs.txt

```diff
@@ -4,4 +4,6 @@ ipython[notebook]
 ipython_genutils
 
 sphinx-autobuild
-https://github.com/PyTorchLightning/lightning_sphinx_theme/archive/master.zip#egg=pt-lightning-sphinx-theme
+pt-lightning-sphinx-theme @ https://github.com/Lightning-AI/lightning_sphinx_theme/archive/master.zip
+
+-r ../../_notebooks/.actions/requirements.txt
```

Comment:
What is this supposed to do?

Reply:
install packages used to build tutorials: https://pytorch-lightning.readthedocs.io/en/latest/tutorials.html?highlight=tutorials